src/main/resources/odf1.3/OpenDocument-v1.3-part2-packages_odt/styles.xml

```diff
@@ -22,7 +22,7 @@
  </office:font-face-decls>
  <office:styles>
   <style:default-style style:family="graphic">
-   <style:graphic-properties draw:stroke="solid" svg:stroke-width="0.071cm" svg:stroke-color="#385d8a" draw:fill-color="#4f81bd" fo:wrap-option="no-wrap" draw:shadow-offset-x="0.3cm" draw:shadow-offset-y="0.3cm" draw:start-line-spacing-horizontal="0.283cm" draw:start-line-spacing-vertical="0.283cm" draw:end-line-spacing-horizontal="0.283cm" draw:end-line-spacing-vertical="0.283cm" style:flow-with-text="true"/>
```

Comment:
but this should be valid?

Reply:
Keep in mind, when you are reviewing commit by commit some changes might be added in the end.
For instance, the exchange of the invalid ODF XML attributes occurred very late in the commit chain.
If you want to see the complete difference from the start to the end of the patch, you need to use some other GitHub URL (I do not have at hand atm)...